python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -215,10 +220,7 @@ def setDefaults(self):
         self.detection.reEstimateBackground = False
         self.detection.thresholdType = "pixel_stdev"
         self.detection.excludeMaskPlanes = ["EDGE",
-                                            "SAT",
                                             "BAD",
-                                            "INTRP",
-                                            "NO_DATA",
```

Comment:
Minor point, but please move the `excludeMaskPlanes` changes to their own commit